### ISSUE TITLE
What is the value of gradient clipping norm you used in the paper?

### ISSUE THREAD

**Issue by Jyouhou** [microsoft/Swin-Transformer#57]
Hi, thanks for this great work!

I noticed a discrepancy between this repo and the paper:

(1) In your paper you said: 'gradient clipping with a max norm of 1 are used' (A2.1.)
(2) In your code and the training log, it looks like a max norm of 5 is used instead. 

What is the correct value to use? Will both work? It seems like the grad norm scarcely exceeds 5 (but almost always above 1), though.

**Comment by ancientmooner** [comment]
They both work. Please follow the settings in this repo which can exactly reproduce the models provided.
